py3status/modules/README.md

```diff
@@ -295,9 +302,9 @@ Obsolete configuration parameters:
     *(default None)*
 
 Requires:
-  - the `acpi` command line
+  - the `acpi` command line (only if `measurement_mode='acpi'`)
 
-**author** shadowprince, AdamBSteele, maximbaz, 4iar
+**author** shadowprince, AdamBSteele, maximbaz, 4iar, m45t3r
```

Comment:
I couldn't run this command. I get "FileNotFoundError", so I just fixed the docstring. Hope someone can fix the README for me.